File: Assignment_1/Assignment1_8.py
"""
-----------------------------------------------------------------------------------------
                            Assignment1_8
                    (Student name - Vaishali Jorwekar)
                     Python by Marvellous Infosystems
-----------------------------------------------------------------------------------------
Problem statement :
    This program accepts number from user and prints * on the console 
------------------------------------------------------------------------------------------"""
STAR="*"
#-----------------------------------------------------------------
# This function accepts a parameter int 'num' from user
# and outputs number of * == num value
#-----------------------------------------------------------------
def PrintStars(num):
    for i in range(0,num):
        if i!=(num-1):
            print(STAR,end="\t")
        else: 
            print(STAR)                
#------------------------------------------------------------------------------
# This function accepts a number  from user and calls PrintStars(num) function
#-------------------------------------------------------------------------------
def main():
    print("---------------------------------------------------")
    print("Enter the number of stars to display : ",end="")
    number = int(input())          # Accept input and conversion to int
    PrintStars(number)     
#------------------------------------------------------------------  
# Entry point of program
#------------------------------------------------------------------
if __name__=="__main__":
    main()
#-----------------------------------------------------------

File: JavaPractice/Assignment_1/Assignment1_8.java
/*--------------------------------------------------------------------------------------
                            Assignment1_8
                    (Student name - Vaishali Jorwekar)
                     Python by Marvellous Infosystems
-----------------------------------------------------------------------------------------
Problem statement :
    This program accepts number from user and prints * on the console 
----------------------------------------------------------------------------------------- */
import java.util.Scanner;

public class Assignment1_8 {
    static char STAR='*';
    public static void printStars(int number){
        for(int cnt=0;cnt<number;cnt++)
            System.out.print(STAR+" ");
        System.out.println();    
    }

    public static void main(String[] args) {
      Scanner scanObj = new Scanner(System.in);
      System.out.print("Enter how many * you want to print:"); 
      int number =scanObj.nextInt();
      printStars(number);
      scanObj.close();
    }
    
}
